docs/examples/affinity/cookie/README.md

```diff
@@ -57,14 +59,16 @@ Last-Modified: Tue, 24 Jan 2017 14:02:19 GMT
 ETag: "58875e6b-264"
 Accept-Ranges: bytes
 ```
-In the example above, you can see a line containing the 'Set-Cookie: INGRESSCOOKIE' setting the right defined stickiness cookie.
-This cookie is created by NGINX, it contains the hash of the used upstream in that request and has an expires. 
-If the user changes this cookie, NGINX creates a new one and redirect the user to another upstream.
 
-If the backend pool grows up NGINX will keep sending the requests through the same server of the first request, even if it's overloaded.
+In the example above, you can see that the response contains a `Set-Cookie` header with the settings we have defined.
+This cookie is created by NGINX, it contains the hash of the upstream used for that request and has an `Expires`.
```

Comment:
>  it contains the hash of the upstream used for that request

This is from old implementation. There's no hash anymore. Following would be better description of what happens now:

> it contains a randomly generated key corresponding (using consistent hashing algorithm) to the upstream used for that request